### PR TITLE
Sessions  Feedback

### PR DESCRIPTION
**Pull request by kavinvalli**
https://github.com/user-attachments/assets/d2dde0bb-9fca-48cc-825f-2e491f097cb4

### CODE REVIEW

**Comment by kavinvalli** [comment]
@chitalian @maamalama I feel like this is good enough to merge now, filters is i think big enough to create a separate pr on

**Comment by kavinvalli** [comment]
3f8404fb3a8e3a3c411a154985cd7a186acb29bc and 199eccada28a7897f72e561d7143d8ff047fe46e implements session feedback
- Implementation Details
    - Creates a property on the first request of the sessions (`Helicone-Session-Feedback`) and sets it to 1 or 0 depending on the feedback
    - In the frontend, I just find the request with the property and set that to the value (suggested by Stefan)
        - The reason, I didn't just get the first request in the frontend as well was because the sorting was working a little differently for some reason and was giving a different request id

https://github.com/user-attachments/assets/6059685c-df08-4042-af90-bf0a9b8afab3

File: valhalla/jawn/src/managers/SessionManager.ts [hunk 1]
```diff
@@ -364,4 +365,46 @@ WHERE ${buildWhereClause("duration")}
...
+    sessionId: string,
+    rating: boolean
+  ): Promise<Result<null, string>> {
+    const query = `
+    SELECT request_id
+    FROM request_response_rmt
+    WHERE properties['Helicone-Session-Id'] = '${sessionId}'
+    ORDER BY request_created_at ASC
+    LIMIT 1
+    `;
```
[the remark this answers is not in the document]
Comment (chitalian) [reply valhalla/jawn/src/managers/SessionManager.ts:379]: Actually can we select from postgres instead? We have better indexes on the JSONB column for custom properties and clickhouse is notoriously bad at point queries

File: valhalla/jawn/src/managers/SessionManager.ts [hunk 2]
```diff
@@ -364,4 +365,46 @@ WHERE ${buildWhereClause("duration")}
...
+
+    console.log({ query });
```
Comment (chitalian) [inline valhalla/jawn/src/managers/SessionManager.ts:381]: rm please

Comment (chitalian) [reply valhalla/jawn/src/managers/SessionManager.ts:381]: and the other console logs below too

**Comment by kavinvalli** [comment]
@chitalian This has too many conflicts and issues. Created a new PR from a new branch instead #2772

**Pull request closed without merging**